src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -504,6 +514,7 @@
       <MonoAOTCompilerDefaultAotArguments Include="static" />
       <MonoAOTCompilerDefaultAotArguments Include="direct-icalls" />
       <MonoAOTCompilerDefaultAotArguments Include="deterministic" />
+      <MonoAOTCompilerDefaultProcessArguments Include="--wasm-exceptions" Condition="'$(WasmExceptionHandling)' == 'true'" />
```

Comment:
we should consider trying to normalize the naming scheme here, for example making it ```--wasm-exception-handling``` to match the msbuild property and be closer to the wasm-opt switch

Reply:
This is just an internal option which will hopefully go away in the future.